src/fabric/src/fabric2_server.erl

```diff
@@ -42,6 +42,21 @@
 -define(FDB_DIRECTORY, fdb_directory).
 -define(FDB_CLUSTER, fdb_cluster).
 -define(DEFAULT_FDB_DIRECTORY, <<"couchdb">>).
+-define(TX_OPTIONS_SECTION, "tx_options").
```

Comment:
I'd rename this to be `fdb_tx_options` so that its clear its about the FDB transaction vs something in CouchDB.

Reply:
agree, good idea